_posts/newsletters/2024-05-28-newsletter.md

```diff
@@ -0,0 +1,349 @@
+* [Code for Thought: Collaborations Workshop 2024](https://codeforthought.buzzsprout.com/1326658/15045013-en-collaborations-workshop-2024) by Peter Schmidt
+* [Connecting research computing, data & software professionals](https://coco.cyberinfrastructure.org/) by Community of Communities
+*  [Eleven strategies for making reproducible research and open science training the norm at research institutions](https://elifesciences.org/articles/89736) by eLife
+*  [Ten simple rules for pushing boundaries of inclusion at academic events](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011797) by Siobhan Mackenzie Hall, Daniel Kochin, Carmel Carne, Patricia Herterich, Kristen Lenay Lewers, Mohamed Abdelhack, Arun Ramasubramanian, Juno Felecia, Michael Alphonse, Visotheary Ung, Sara El-Gebali, Christopher Brian Currin, Esther Plomp, Rachel Thompson, Malvika Sharan  
+*  [Inclusive Language in Technology](https://www.aswf.io/blog/inclusive-language/) by Academy Software Foundation
+*  [Racial Equity Tools Glossary](https://www.racialequitytools.org/glossary) by Racial Equity Tools
+*  [There’s no such thing as sustainable research software](https://danielskatzblog.wordpress.com/2024/05/13/no-sustainable-research-software/) by Daniel S. Katz
+
+-----------------
+
+<a name="financials"></a>
+# **7. US-RSE Financials**
+
+From a financial perspective, 2023 was an exciting year for US-RSE. We were awarded a two-year grant from the Alfred P. Sloan Foundation and switched fiscal sponsors. We are very grateful to the Open Collective Foundation (OCF) that was our fiscal sponsor until last spring. However, with the awarded grant our needs changed and we are now a fiscally sponsored project of [Community Initiatives](https://communityinitiatives.org/) (CI). CI gives us the structure and resources to hire staff such as the Executive Director and the Community Manager, accept donations and reimburse members for expenses they incurred on behalf of US-RSE. The community funds possible via the Alfred P. Sloan grant are also awarded via CI.
+
+The majority of the grant from the Sloan Foundation is budgeted to pay for our new Executive Director and Community Manager. Besides that we've budgeted money to support staff travel, the US-RSE conference, fund community activities through our new Community and Travel Fund program, and fund smaller items such as code of conduct trainings for our code of conduct committee members. CI charges 10% of our income for providing resources, infrastructure, and support (trust us when we say we make good use of those resources). Switching from OCF to CI turned out to been an unexpectedly complex change and it has been a very steep learning curve. We appreciate our members' patience as we navigate the world of non-profit regulations and guidelines.  
+
+-----------------
+<a name="#sc-update"></a>
+# **8. Steering Committee Recap**
```

Comment:
```suggestion
# **8. Steering Committee Update**
```

Reply:
In the circle preview, this link doesn't work. Not sure why!?!

Reply:
I think I figured it out - there was a # that wasn't needed in the <a name...>. Suggested a change above